src/components/pool_flux_chart.vue

```diff
@@ -32,14 +33,17 @@
           <label><input type="radio" name="x-scale" value="log" checked> log </label>
           <label><input type="radio" name="x-scale" value="linear"> linear </label>          
         </form>
-        <p>You are probably familiar with linear scales. When you move a fixed distance on a linear axis, you add a fixed value to your starting value. When you move a fixed distance on a log axis, you multiply the starting value by a value of 10.</p>
+        <p :text="axisExplanation">{{ axisExplanation }}</p>
+        <p><span class='poolText' >Pools</span> are places where water is stored, like the ocean. <span class='fluxText'>Fluxes</span> are the ways that water moves between pools, such as evaporation, precipitation, discharge, recharge, or human use.</p>
```

Comment:
What about moving this text about pools and fluxes to the top under the title?

Reply:
I had it there, but wasn't sure if it bumped the chart too far down the page. But now I think it looks nice:
![image](https://user-images.githubusercontent.com/54007288/191867085-82c7b7c2-c8c2-4241-b300-33d2322e3602.png)

I'll push those changes

Reply:
Might also be nice to use the text color for the title, and to make that title bold

Reply:
Hah I had that and reverted it. I'll add it back in.